virtcontainers/qemu.go

```diff
@@ -64,6 +64,9 @@ const qmpCapErrMsg = "Failed to negoatiate QMP capabilities"
 
 const defaultConsole = "console.sock"
 
+var qemuMajorVersion int
+var qemuMinorVersion int
```

Comment:
I wonder if we want to store these in one of the types, even though they are only being used for PPC64le atm.

Reply:
@jodh-intel : Should we? I am not sure if its gonna get consumed by any other architectures atm.

Reply:
I would definitely include those as part of the `qemu` structure since this is an info about the hypervisor. And I agree with @jodh-intel that even if it's only used by ppc64, this is not a reason to define some global variable there.

Reply:
@sboeuf : Okay, I shall add qemuMajorVersion and qemuMinorVersion into the qemu structure. However, I am unsure of what would be the most efficient way of exposing it in `func (q *qemuPPC64le) memoryTopology`.  One way is to get qemu version info in `func (q *qemu) init(sandbox *Sandbox) error` and pass via `newQemuArch` resulting in change of signature of `newQemuArch` everywhere.

Reply:
It's a bit fiddly, but we could add them to `qemuArchBase` like this:

- https://github.com/jodh-intel/runtimes/tree/set-and-get-qemu-version.

Reply:
@nitkon oh yeah my bad, I forgot `qemu` structure was not the interface related to `qemu_ppc64le`. As @jodh-intel suggested, `qemuArchBase` should be the one.

@jodh-intel, I have taken a look at your suggestion but I thought we could simplify this by simply having those fields being called from `qemu_ppc64le` structure (no need for getters and setters). Isn't it possible since this structure inherits from `qemuArchBase` ?

Reply:
I'm not quite sure what you're saying tbh. I'm hoping there is a simpler way though as I'm not particularly happy with what I came up with :)

Reply:
@jodh-intel : So I was going through the implementation but noticed the following.
`Printing q.qemuMajorVersion, q.qemuMinorVersion in "func (q qemuArchBase) getQEMUVersion() (major, minor int) {" shows me the following:
level=info msg="DEBUG: versions: major: 0, minor: 0" arch=amd64 name=kata-runtime pid=6215 .`

Reply:
@jodh-intel : I am a bit confused by the reference implementation. We are setting qemuMajorVersion and qemuMinorVersion using one qemu instance and getting it with another qemu instance. (resulting it in 0,0 values). 

IIUC, we need to set/get using the same qemu instance